lang/src/org/partiql/lang/ast/passes/AstVisitor.kt

```diff
@@ -27,7 +27,6 @@ import org.partiql.lang.ast.*
  *
  *  One `visit*` function is included for each base type in the AST.
  */
-@Deprecated("Use AstNode#iterator() or AstNode#children()")
```

Comment:
Well, I removed this one because it was providing information that contradicted the `@Deprecated` tags I added earlier in this PR. However, it should probably say: "Use org.lang.partiql.domains.PartiqlAst.Visitor instead"` instead, which will be in the next commit.